l10nfr.properties

```diff
@@ -27,7 +27,13 @@ defaults.mavenArtifactId=sonar-l10n-fr-plugin
 9.8.1.changelogUrl=https://github.com/jycr/sonar-l10n-fr/releases/tag/9.8.1
 
 9.9.0.description=Support SonarQube 9.9+
-9.9.0.sqVersions=[9.9,LATEST]
+9.9.0.sqVersions=9.9
```

Comment:
You can't have two public versions with overlapping compatibility. The Marketplace has to know which plugin version to recommend for each SQ version.
I suggest you archive this one and set its compatibility to: `[9.9,9.9.*]` so that users on 9.9.0 will be prompted to upgrade

Reply:
Thanks for your advice